build/charts/antrea/crds/externalentity.yaml

```diff
@@ -44,6 +44,9 @@ spec:
                 externalNode:
                   type: string
     - name: v1alpha1
+      deprecated: true
```

Comment:
There are a few more CRDs with two or more served versions, I think we need to add this field for all old versions. @antoninbas what's your thoughts?

Reply:
Yes, we have more deprecated CRD versions according to https://github.com/antrea-io/antrea/blob/main/docs/api.md#crds-in-crdantreaio

Looking at this document, I have 2 questions though:
- why is `v1alpha1/ExternalEntity` not listed in the doc?
- the doc mentions that `v1alpha1/Tier` is supposed to be removed in Antrea v2.0. Are we still planning to do that?

Reply:
> There are a few more CRDs with two or more served versions, I think we need to add this field for all old versions. @antoninbas what's your thoughts?

Yes, that is what I want to ask. So, if one CRD has more than one served version, we just keep the latest one and add `deprecated: true` and deprecation warning to other versions?.

Reply:
Yes. https://github.com/antrea-io/antrea/blob/main/docs/api.md#crds-in-crdantreaio should also be up-to-date in that regard (see `Deprecated in` column).

Reply:
Updated.

Reply:
Pinging @luolanzone for the questions above: https://github.com/antrea-io/antrea/pull/6089#discussion_r1539749892

Reply:
I noticed that `v1alpha1/ExternalEntity` is not served, so can we just remove this APIVersion, cause the `deprecated` filed will not work on this CRD.

Reply:
I checked the doc history, seems we didn't add  v1alpha1/ExternalEntity at the very beginning in this doc.
As @hjiajing  mentioned, it's not served and I can only see following content in crds.yml without any schema. I believe we can simply remove it in this PR.
```
    - name: v1alpha1
      served: false
      storage: false
      schema:
        openAPIV3Schema:
          type: object
```
For `v1alpha1/Tier`, I think we can remove it. I will handle this.

Reply:
The PR is submitted to remove the v1alpha1/Tier https://github.com/antrea-io/antrea/pull/6162
@hjiajing you need to rebase after the merge. Thanks.

Reply:
Thanks for the reminder, I will rebase after that PR is merged.

Reply:
Hi @antoninbas , I created a PR to remove this CRD https://github.com/antrea-io/antrea/pull/6177

Reply:
Thanks. I assume this PR will need to be rebased after we merge #6177